tests/Rules/Locale/TurkishCharacterTest.php

```diff
@@ -0,0 +1,19 @@
+<?php
+namespace Respect\Validation\Rules\Locale;
+
+use Respect\Validation\Rules\Locale;
+
+class TurkishCharacterTest extends \PHPUnit_Framework_TestCase
+{
+    public function testShouldAcceptTurkish()
+    {
+        $rule = new TurkishCharacter;
+        $this->assertTrue($rule->validate('Mustafa Kemal ATATÜRK'));
```

Comment:
Could you use some data providers? Then you can test more situations, like camel case, for example.